cirq-core/cirq/study/flatten_expressions.py

```diff
@@ -337,9 +337,9 @@ def transform_sweep(
             sweep: The sweep to transform.
         """
         sweep = sweepable.to_sweep(sweep)
-        param_list = []
+        param_list: List[Dict[Union[str, sympy.Expr], Union[float, sympy.Expr]]] = []
```

Comment:
We have an existing type alias for these parameter dicts
```suggestion
        param_list: List['cirq.ParamDictType'] = []
```
This comes up in a number of places in this PR.